tests/test_spam_mixin.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import absolute_import
+from datetime import datetime
+import mock
+from nose.tools import *  # noqa PEP8 asserts
+from modularodm.exceptions import ValidationValueError, ValidationTypeError
+
+from framework.auth import Auth
+
+from tests.base import OsfTestCase
+from tests.factories import UserFactory, CommentFactory
+
+from website.project.model import SpamMixin
+
+
+class TestSpamMixin(OsfTestCase):
+
+    def setUp(self):
+        super(TestSpamMixin, self).setUp()
+        self.comment = CommentFactory()
+        self.auth = Auth(user=self.comment.user)
+
+    @mock.patch('website.project.model.SpamMixin.flag_spam')
+    def test_report_abuse(self, mock_flag_spam):
+        user = UserFactory()
+        time = datetime.utcnow()
+        self.comment.report_abuse(
+                user, date=time, category='spam', text='ads', save=True)
+        equivalent = dict(
+            date=time,
+            category='spam',
+            text='ads',
+            retracted=False
+        )
+        assert_in(user._id, self.comment.reports)
+        assert_equal(self.comment.reports[user._id], equivalent)
+        assert_equal(len(mock_flag_spam.mock_calls), 1)
+
+    @mock.patch('website.project.model.SpamMixin.flag_spam')
```

Comment:
I don't think the mocking is necessary. Instead, you can assert that spam_status is set to the correct value.